### PR TITLE
Week 4 Blogpost

### PR DESCRIPTION
**Pull request by JoaoDell**
Hello guys, here is my week 4 blogpost:
image

### CODE REVIEW

File: docs/source/posts/2023/2023-06-26-week-4-joaodellagli.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+.. post:: June 26, 2023
+   :author: João Victor Dell Agli Floriano
+   :tags: google
+   :category: gsoc
+
+
+Welcome again to another weekly blogpost! Today, let's talk about the importance of guidance throughout a project.
+
+This Last Week's Effort
```
Comment (ganimtron-10) [inline docs/source/posts/2023/2023-06-26-week-4-joaodellagli.rst:12]: This sentence seem a bit odd to me.
You can either use `This Week's Effort` or `Last Week's Effort`.

File: docs/source/posts/2023/2023-06-26-week-4-joaodellagli.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Here, I would like to point out how my mentor was **essential** to this solution to come: I had struggled for some time with that, and could 
+not find a way out, but a single session of synchronous pair programming where I could expose clearly my problem and talk to someone 
+way more experienced than I, someone designated for that, was my way out of this torment, so value your mentors! Thanks Bruno!
+
+
+This Week's Goals
+-----------------
+Now, with the FBO working, I plan to finally *render* something to it. For this week, I plan to come back to my original plan and
+experiment with simple shaders just as a a proof of concept that the FBO will be really useful for this project. I hope the road is less
```
Comment (ganimtron-10) [inline docs/source/posts/2023/2023-06-26-week-4-joaodellagli.rst:57]: `a` repeated twice in `just as a a proof`

**Review by ganimtron-10 — CHANGES_REQUESTED** [review]
Hey @JoaoDell,
You blogpost generated well locally. I just had the below comments!

File: docs/source/posts/2023/2023-06-26-week-4-joaodellagli.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Where the Problem Was
+---------------------
+Apparently, for the FBO generation to work, it is first needed to initialize the context interactor:
+
+::
+
+   FBO = vtk.vtkOpenGLFramebufferObject()
+
+   manager.window.SetOffScreenRendering(True) # so the window don't show up, but important for later as well
```
Comment (tvcastillod) [inline docs/source/posts/2023/2023-06-26-week-4-joaodellagli.rst:29]: `don't` => `doesn't`

**Review by tvcastillod — COMMENTED** [review]
Hi @JoaoDell, glad you were able to fix the problem, great teamwork!
I just have one minor comment, but everything else looks good to me.

**Comment by JoaoDell** [comment]
@ganimtron-10 @tvcastillod thanks for the reviews, just changed what you pointed out :)

**Review by skoudoro — APPROVED** [review]
Looks good to me, thanks to the reviewers and @JoaoDell.

small note for the future blog post: Try to do shorter sentences. Check general literature in english. Short sentences make it easier to read. The "last week effort" section is a bit hard read.  

Merging

**Pull request merged**